test/unit/mixins/state.spec.js

```diff
@@ -1,9 +1,19 @@
-describe('State Class', function() {
+import Marionette from 'backbone.marionette';
+import StateMixin from '../../../src/mixins/state';
```

Comment:
I think you should be able to use `Marionette.Toolkit.StateMixin` instead of importing it.